### PR TITLE
Inform the test requirements

### PR DESCRIPTION
**Pull request by alanjds**
Inform the stuff needed to test on ``setup.py``

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def run(self):
 
         try:
             os.environ['EPYTHON'] = 'python{}.{}'.format(sys.version_info.major, sys.version_info.minor)
-            subprocess.check_call(['py.test', '-v', testpath,
+            subprocess.check_call(['py.test', '-v', testpath, '-s',
```
Comment (otoolep) [inline setup.py]: Why did you make this change?

Comment (alanjds) [reply setup.py]: Oops. It slipped thru.

Made to allow the STDIN/STDOUT to go unmodified thru py.test until the end of the test. Is the default here around me, but should not had slipped on this unrelated PR.

Comment (alanjds) [reply setup.py]: I can revert this if you want.

Comment (otoolep) [reply setup.py]: Yeah, let's revert for now and I'll merge otherwise -- thanks.

**Comment by otoolep** [comment]
Thanks @alanjds !

**Pull request merged**